recipes/mrcz/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "mrcz" %}
+{% set version = "0.3.6" %}
+{% set sha256 = "8b97aa55d2827d0d8371f5a713d6c26a68952c97631a2243b698e0fcb4998992" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+    - numpy
+    - futures  # [py2k]
+  run:
+    - python
+    - numpy
+    - blosc
+    - futures  # [py2k]
+
+test:
+  imports:
+    - mrcz
+
+about:
+  home: https://github.com/em-MRCZ/python-mrcz
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```

Comment:
To fix this long term, issue ( https://github.com/em-MRCZ/python-mrcz/issues/10 ) has instructions to ensure the license file is added to `sdist`s uploaded to PyPI, which we can use in the future.